.github/workflows/examples-nrfconnect.yaml

```diff
@@ -45,12 +45,18 @@ jobs:
             #   with:
             #       languages: "cpp, python"
             #       queries: +security-and-quality
-            - name: Build example nRF Connect SDK Lock App
+            - name: Build example nRF Connect SDK Lock App on nRF52840
               run: scripts/examples/nrfconnect_example.sh lock-app
-            - name: Build example nRF Connect SDK Lighting App
+            - name: Build example nRF Connect SDK Lighting App on nRF52840
               run: scripts/examples/nrfconnect_example.sh lighting-app
-            - name: Build example nRF Connect SDK Shell
+            - name: Build example nRF Connect SDK Shell on nRF52840
               run: scripts/examples/nrfconnect_example.sh shell
+            - name: Build example nRF Connect SDK Lock App on nRF5340
```

Comment:
Not sure.. if we want a new job for nRF5340, or build both nRF52840 and nRF5340 when no parameters are provided to the script. That's something up to us i guess.

Reply:
I would say a single script invocation should build a project for a single board. Note the script is also used by VSCode commands. Btw, do you think we need to verify build for all apps and all boards in CI? Or maybe we should check a single app, say, lighting-app for nRF53 and the rest in our nightly run? Faster feedback from CI speeds up development time, so it's worth to weigh in pros and cons. But maybe we're not at the point where we should be concerned about that?

Reply:
I agree with the statement of the advantage of having a fast CI.

In the other hand right now, our checks are far from the longest one, plus if we implement your suggestion to remove bootstrap, we might save another 2-3 minutes. So I don't think this should be our concern now. @kkasperczyk-no what do you think? Based on this, I would prefer to block others to create regression on our boards as much as we can.

I would actually even add lightning-app/pigweed alternative into CI as well :smile:. But this is just my thoughts.

Reply:
Ok, let's leave it for now :) And yes, I thought about adding light-app with RPC, too. We should probably do that.

Reply:
Let's add it in different PR.